core/tests/puppeteer-acceptance-tests/spec/voiceover-admin-tests/add-voiceover-artist-to-an-exploration.spec.ts

```diff
@@ -0,0 +1,110 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utility functions for voiceover admin page if voiceover admin
+ * can add voiceover artist to an exploration
+ */
+
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import {VoiceoverAdmin} from '../../user-utilities/voiceover-admin-utils';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {ConsoleReporter} from '../../puppeteer-testing-utilities/console-reporter';
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+const ROLES = testConstants.Roles;
+const invalidIdErrorToastMessage =
+  'Sorry, we could not find the specified user.';
+
+ConsoleReporter.setConsoleErrorsToIgnore([
+  new RegExp(
+    'http://localhost:8181/voice_artist_management_handler/exploration/.*Failed to load resource: the server responded with a status of 400'
+  ),
+  new RegExp('Sorry, we could not find the specified user.'),
+]);
+
+describe('Voiceover Admin', function () {
+  let voiceoverAdm: VoiceoverAdmin;
+  let explorationEditor: VoiceoverAdmin;
+  let explorationId: string | null;
+
+  beforeAll(async function () {
+    voiceoverAdm = await UserFactory.createNewUser(
+      'voiceoverAdm',
+      'voiceover_admin@example.com',
+      [ROLES.VOICEOVER_ADMIN]
+    );
+
+    explorationEditor = await UserFactory.createNewUser(
+      'explorationEditor',
+      'exploration_creator@example.com'
+    );
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it(
+    'should be able to see error while adding an invalid user as a voiceover artist to an exploration',
+    async function () {
+      await explorationEditor.navigateToCreatorDashboardPage();
+      await explorationEditor.navigateToExplorationEditorPage();
+      await explorationEditor.dismissWelcomeModal();
+
+      await explorationEditor.createExplorationWithTitle('Exploration one');
+      explorationId =
+        await explorationEditor.publishExplorationWithTitle('Exploration one');
+
+      await voiceoverAdm.navigateToExplorationEditor(explorationId);
+      await voiceoverAdm.dismissWelcomeModal();
+
+      await voiceoverAdm.navigateToExplorationSettingsTab();
+
+      await voiceoverAdm.expectVoiceoverArtistNotExists('invalidUserId');
+      await voiceoverAdm.addVoiceoverArtistToExploration('invalidUserId');
+
+      await voiceoverAdm.expectToSeeErrorToastMessage(
+        invalidIdErrorToastMessage
+      );
+      await voiceoverAdm.closeToastMessage();
```

Comment:
Maybe add one more check to confirm the voiceover artist still doesn't show up in the list.

Reply:
Done